tests/lean/interactive/hover.lean

```diff
@@ -175,6 +175,9 @@ example : Nat → Nat → Nat :=
     x
   --^ textDocument/hover
 
+           -- textDocument/definition -- removed because the result is platform-dependent
```

Comment:
Unfortunately this is another of those tests that can't be written: This test jumps into the location of the lean 4 sources, which is different for the different CI runners and also for me locally. The goTo.lean tests fix this by only doing intra-file jumps, but options must be declared in a separate file so we have the usual issue about tests being single-file only.

Reply:
Can we just replace the lean4 source directory by the empty string like we do for the tests?

Reply:
I don't know if this is desirable. We try to avoid source location references into src/ from tests since they are far from stable. It's really annoying when supposedly unrelated tests suddenly fail.